src/main/java/sg/ncl/MainController.java

```diff
@@ -104,10 +105,10 @@ public String plan() {
         return "plan";
     }
 
-    @RequestMapping("/futureplan")
-    public String futureplan() {
-        return "futureplan";
-    }
+//    @RequestMapping("/futureplan")
+//    public String futureplan() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)